### PR TITLE
Final preparation for v0.3 release

### PR DESCRIPTION
**Pull request by rn**
- Update all packages and tests to use the latest alpine base
- Bump version number
- Add changelog

Once merged, I'll tag a release and upload the binaries.

image

### CODE REVIEW

**Comment by rn** [comment]
CI failed because one of the images (`linuxkit/init`) didn't have the tag `v0.3` applied. Will forcepush and then hit re-test on the CI

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,6 +33,12 @@ jobs:
             GOOS: linux
             GOARCH: arm64
           command: make LOCAL_TARGET=bin/linuxkit-$GOOS-$GOARCH local-build
+      - run:
+          name: Build arm64/linux
```
Comment (justincormack) [inline .circleci/config.yml]: that should read s390x linux

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## [v0.2] - 2018-01-25
+- Almost everything
+
+
+## [v0.1] - 2017-??-??
+- Sometime in 2017 we did a mini v0.1 release but we seem to have lost any trace of it :)
+
+
+## [v0.0] - 2017-04-18
+- Initial open sourceing of LinuxKit
```
Comment (justincormack) [inline CHANGELOG.md]: typo "sourcing"

**Comment by rn** [comment]
force pushed an update to the CHANGELOG but added an additional commit to fix the name for the CircleCI build step so that I don't need to rebuild all the packages because the git commits in the meta data would have changed

**Pull request merged**